kubeflow/kpt-set.sh

```diff
@@ -7,10 +7,10 @@ kpt cfg set -R .  asm-label ${ASM_LABEL}
 kpt cfg set -R .  mgmt-ctxt ${MGMT_NAME}
 
 kpt cfg set -R .  name ${KF_NAME}
-kpt cfg set -R .  gcloud.project.projectNumber <KF_PROJECT_NUMBER>
+kpt cfg set -R .  gcloud.project.projectNumber ${KF_PROJECT_NUMBER}
```

Comment:
nit: always surround variable with ""

E.g. "${KF_PROJECT_NUMBER}", because there can be spaces that make the script fail in unexpected ways

Reply:
It is a very good point, thank you for mentioning! I will add them in shell scripts. 

For now I will patch them in https://github.com/kubeflow/gcp-blueprints/pull/237/ for convenient.